core/tests/test_office.py

```diff
@@ -48,20 +72,22 @@ def test_cannot_add_existing_office_block(self):
                     name="Block A"
                 )
                 new_office_block.save()
-
         self.assertEqual(self.all_office_blocks.count(), 1)
 
     def test_office_block_model_string_representation(self):
         self.assertEqual(str(self.office_block), "Block A")
 
     def test_office_floor_model_string_representation(self):
-        self.assertEqual(self.number.number, 5)
+        self.assertEqual(self.office_floor.number, 5)
 
-    def test_add_new_office_floor(self):
-        """Test add new floor section"""
-        self.assertEqual(self.floor_number_counts.count(), 1)
-        new_office_floor = OfficeFloor(
-            number=10,
-            block=self.office_block)
-        new_office_floor.save()
-        self.assertEqual(self.floor_number_counts.count(), 2)
+    def test_office_floor_section_model_string_representation(self):
+        self.assertEqual(str(self.office_floor_section), "Right Wing")
+
+    def test_can_add_existing_office_floor_section(self):
```

Comment:
Isn't this a repetition. Did you mean to test that a duplicate office floor cannot be created?
https://github.com/AndelaOSP/art-backend/blob/36232fb08b4e1d979f5c1f65e25c0b9282f1b4cb/core/tests/test_office.py#L57-L64

Reply:
@QUDUSKUNLE Update the last test case to check for expected behaviour when there's a duplicate entry on the **OfficeFloorSection** model

Reply:
It is not a repetition. To test a duplicate `office_section` on the same floor.

Reply:
From what I can tell; `test_add_new_office_floor_section()` is checking for the creation behaviour on the OfficeFloorSection model, with the `floor` FK pointing to `self.office_floor`. Same thing for `test_can_add_existing_office_floor_section()`. Bear in mind that `OfficeFloorSection.objects.create()` and `OfficeFloorSection()` both create an object based-off the **OfficeFloorSection** model.

Reply:
Yes, pointing to the same floor but different `floor_section`. Initially, it was `Right Wing` but now `Left Wing`.

Reply:
Both **OfficeFloorSection**s are named "Left Wing".
https://github.com/AndelaOSP/art-backend/blob/36232fb08b4e1d979f5c1f65e25c0b9282f1b4cb/core/tests/test_office.py#L61
https://github.com/AndelaOSP/art-backend/blob/36232fb08b4e1d979f5c1f65e25c0b9282f1b4cb/core/tests/test_office.py#L90